desk/lib/channel-utils.hoon

```diff
@@ -29,13 +29,19 @@
   %-  ~(run by v-channels)
   |=  channel=v-channel:c
   ^-  channel:c
-  %*  .  *channel:c
-    posts  ?:(full (uv-posts posts.channel) *posts:c)
-    perm   +.perm.channel
-    view   +.view.channel
-    sort   +.sort.channel
-    order  +.order.channel
-    pending  pending.channel
+  =/  base
+    %*  .  *channel:c
+      perm   +.perm.channel
+      view   +.view.channel
+      sort   +.sort.channel
+      order  +.order.channel
+      pending  pending.channel
+    ==
+  ?.  full  base
+  %*  .  base
+    posts  (uv-posts posts.channel)
+    net  net.channel
+    remark  remark.channel
   ==
```

Comment:
This is just `%_  base` (in place of `%*  .  base`). We need it for `*channel:c` because that needs to be evaluated, and we modify the result, but here `base` already is a noun in our subject, so we can just modify that directly.